src/main/java/org/folio/oaipmh/helpers/storage/InventoryStorageHelper.java

```diff
@@ -42,7 +42,7 @@ public String getIdentifierId(final JsonObject entry) {
   }
 
   @Override
-  public String buildRecordsEndpoint(Request request) throws UnsupportedEncodingException {
+  public String buildRecordsEndpoint(Request request, boolean recordsPath) throws UnsupportedEncodingException {
```

Comment:
For boolean variable the recordsPath is not the appropriate name. So, for me it's confusing to get what this variable means here from logic side. Rename it like isRecordPath.